citi-onboarding/celeiro-dos-bambas
# Feature/contact-session

## What I did

- I added the "**contact-session**" in frontend.
**[Edit]**
- I added `axios` lib.
- I make the adjusts requested for style. 

## How to test

1. Open the terminal and go to the **client** folder of this project

1. In terminal:
   1. `git pull`
   1. `git checkout feature/contact-session`
   1. Remove `<Footer />` in app.jsx
   1. `yarn install`
   1. `yarn start`

1. In your browser go to [http://localhost:1337/admin](http://localhost:1337/admin)

1. Check if the template present is similar with the Design Root. 
1. Test the zoom-in and zoom-out
1. Lock the `forms` in the code.
